src/back/controllers/api/routes/challenge/getOne.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const challenge = require('../../models/challenge');
+
+const getOne = (req, res) => {
+    challenge.findOne({ _id: req.params.id }, (err, challengeDetails) => {
```

Comment:
We should avoid use persistence layer direct in controller.
please create facade for this kind of operations to keep route handlers just interlayer between business logic and web interface